core/templates/pages/about-page/about-page.module.ts

```diff
@@ -66,6 +68,12 @@ import { AboutPageConstants } from 'pages/about-page/about-page.constants';
       provide: HTTP_INTERCEPTORS,
       useClass: RequestInterceptor,
       multi: true
+    },
+    {
+      provide: APP_INITIALIZER,
```

Comment:
Here and elsewhere: how can we make sure that new pages have this present? If it requires relying on people to remember, we should prefer to generalize the solution or find a way to trigger an automatic test failure for those new pages.

Reply:
I think the `HTTP_INTERCEPTORS` above is in a similar situation, it's also presented in every page module. Not sure how it's treated but if there isn't specific test for `HTTP_INTERCEPTORS`, my suggestion would be leaving the tests for these kind of tests as a future work.

Reply:
@srijanreddy98 do you have any thoughts on this? This could end up pretty bad if we miss the initializer in some places: features won't be enabled for those pages.

Reply:
Do we have any new page modules coming up (before November)? If not, then we don't have a problem because we will be migrating to a structure where all these 42 page-modules would be lazy-loaded and we will have a root AppModule where this logic would reside.

Reply:
In the scope of the dynamic feature gating project, I don't think there's any new page module coming, so it should be fine.